knowledge-base/kb-datetimevalues-to-formattedstrings.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Formatting Display of DateTime Values in Selection Box
+description: "How to convert all DateTime values to list of formatted strings."
+type: how-to
+page_title: How to Display DateTime Values as Formatted Strings in Selection Box for WPF
+slug: kb-datetimevalues-to-formattedstrings
+position: 0
+tags: DateTime, AllowMultipleSelection, MultipleSelectionBoxTemplate
+ticketid: 1622029
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadComboBox for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to format `DateTime` values to strings in `SelectionBox`.
+
+## Solution
+
+To get the desired effect, you can bind to the `SelectedItems` or to the original string value and use an `IValueConverter` to format the values.
+
+#### __[C#]__
+{{region kb-datetimevalues-to-formattedstrings}}
+	public class DateTimeStringConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object 	parameter, CultureInfo culture)
+		{
+			var dateStrings = ((string)value).Split(',');
+			var format = parameter.ToString();
+			List<string> result = new List<string>();
+			foreach (string dateString in dateStrings)
+			{
+				var date = DateTime.Parse(dateString, 	CultureInfo.InvariantCulture);
+				result.Add(date.ToString(format, 	CultureInfo.InvariantCulture));
+			}
+			return string.Join(", ", result);
+		}
+	}
+{{endregion}}
+
+
+## Solution
```

Comment:
Why you have two Solution sections in this article? 
I think these two sections should be merged into one.

Also, no need to be so detailed about the TextBlock's Text property. It is enough to tell that the MultipleSelectionBoxTemplate can be used and in of the template you can use an element bound to the SelectedItems with an IValueConverter.